app/views/map/_leaflet.html.erb

```diff
@@ -0,0 +1,8 @@
+<script>
+        var map = L.map('map').setView([<%= lat %>,<%= lon %>], 10);
+        L.tileLayer("https://a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map)
+
+        <% @maps.each do |map| %>
```

Comment:
Here I believe `@maps` should be `items` -- because we want the one passed in from the helper. Make sense? And inside the `each` loop, let's call them "items" not "maps" since we could be displaying different kinds of things on a map, as long as they have: lat, lon, path, and title.